Inventory/Model/ResourceModel/SourceTypeLink.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Inventory\Model\ResourceModel;
+
+use Magento\Framework\Model\ResourceModel\Db\AbstractDb;
+
+/**
+ * Implementation of basic operations for SourceTypeLink entity for specific db layer
+ *
+ * This class needed for internal purposes only, to make collection work properly
+ */
+class SourceTypeLink extends AbstractDb
+{
+    /**#@+
+     * Constants related to specific db layer
+     */
+    const TABLE_NAME_SOURCE_TYPE_LINK = 'inventory_source_type_link';
```

Comment:
i can't because I use lot of time:
https://github.com/TadeuRodrigues/msi/blob/92ca7315979f58908d2af7ac1dddf951898161c5/Inventory/Model/ResourceModel/SourceTypeLink/Save.php#L53
https://github.com/TadeuRodrigues/msi/blob/92ca7315979f58908d2af7ac1dddf951898161c5/Inventory/Model/ResourceModel/SourceTypeLink/Delete.php#L43
https://github.com/TadeuRodrigues/msi/blob/92ca7315979f58908d2af7ac1dddf951898161c5/Inventory/Model/ResourceModel/GetSourceTypeBySourceCode.php#L38